src/include/daos_fs.h

```diff
@@ -44,6 +44,8 @@ extern "C" {
 /** Maximum xattr value */
 #define DFS_MAX_XATTR_LEN	65536
 
+#define DAOS_SUPER_MAGIC        0xDA05AD10
```

Comment:
please fix this doxygen warning by just adding a comment to describe this.
just like /** magic value for stafs */

Reply:
should also use clang format (the daos commit hook can be used with git-clang-format to auto update this for you on commit): https://github.com/daos-stack/daos/tree/master/utils/githooks#readme

```
sudo dnf install git-clang-format -y
git config core.hookspath utils/githooks
```

Reply:
thanks: I've set up all the githooks now..  next pull request should go a lot more smoothly!